x-pack/plugins/maps/public/embeddable/map_embeddable_factory.ts

```diff
@@ -141,7 +141,7 @@ export class MapEmbeddableFactory implements EmbeddableFactoryDefinition {
   ) => {
     await waitForMapDependencies();
     const savedMap = await this._fetchSavedMap(savedObjectId);
-    const layerList = getInitialLayers(savedMap.layerListJSON);
+    const layerList = JSON.parse(savedMap.layerListJSON);
```

Comment:
??

Reply:
Agree with Thomas, why was logic removed from getInitialLayers? What if savedMap.layerListJSON is not provided? Want to at least provide road map.